src/components/Select/index.tsx

```diff
@@ -22,17 +23,24 @@ const Select: FC<Props> = ({
   label = '',
   selectedValue,
   values,
+  id,
   ...rest
 }) => (
   <Wrapper error={error !== ''} {...rest}>
     <InputLabel>{label}</InputLabel>
     <MaterialSelect
+      id={id}
```

Comment:
é opcional, mas não vale a pena por um valor padrão?
No caso do id não passado, o que acontece aqui? 🤔

Reply:
Fica sem ID na DOM, da pra colocar um valor padrão, mas dai muda a DOM das aplicações que usam a lib. Acha que devo colocar?

Reply:
id, pela dom são naturalmente valores opcionais, eu concordo em manter opcional

Reply:
Última dúvida....
no fim das contas, se não for passado id o select vai ficar assim?
```
<select></select>
```
ou assim:
```
<select id="undefined"></select>
```

Reply:
`<select></select>`